pom.xml

```diff
@@ -306,6 +306,9 @@
     <jboss.logging.annotations.version>3.0.3.Final</jboss.logging.annotations.version>
     <jboss.logging.processor.version>3.0.3.Final</jboss.logging.processor.version>
     <jboss.logging.version>3.6.1.Final</jboss.logging.version>
+    <jdk.disable.virtualthreads>[17,20}</jdk.disable.virtualthreads>
```

Comment:
Let's only keep `jdk.enable.*` properties.

You can refer to the opposite in this way:

```xml
      <activation>
        <jdk>!${jdk.enable.virtualthreads}</jdk>
      </activation>
```

Reference: https://maven.apache.org/guides/introduction/introduction-to-profiles.html

Reply:
Unfortunately, the property approach didn't work, so it was reverted.
But the negative activation is still a good addition.

Reply:
Why does it not work?

Reply:
profile activation conditions are evaluated before properties. (because properties from a profile can override previous properties. Maven must know first what properties are available before using them so profile first (almost chicken and egg problem :))